src/config/rcxml.c

```diff
@@ -1064,11 +1064,14 @@ static struct mouse_combos {
 	{ "Title", "Left", "DoubleClick", "ToggleMaximize", NULL, NULL },
 	{ "TitleBar", "Right", "Click", "Focus", NULL, NULL},
 	{ "TitleBar", "Right", "Click", "Raise", NULL, NULL},
-	{ "TitleBar", "Right", "Click", "ShowMenu", "menu", "client-menu"},
+	{ "Title", "Right", "Click", "ShowMenu", "menu", "client-menu"},
```

Comment:
This needs to be reflected in `docs/rc.xml.all` to have it synced with our built-in defaults.

It is a subtle change that we should mention this in the changelog. Which brings me to the next point, we should likely have some pinned issue to note changes like that so we don't forget to mention it.